lmp/tokenizer/_base_dict_tokenizer.py

```diff
@@ -323,9 +360,21 @@ def batch_encode(
                 longer than `max_seq_len` and padded to `max_seq_len` when
                 individual sequence length is shorter than `max_seq_len`.
 
+        Raises:
+            TypeError:
+                When `batch_sequences` is not instance of `List[str]`.
+                When `max_seq_len` is not instance of `int`.
```

Comment:
When `batch_sequences` is not instance of `List[str]` or `max_seq_len` is not instance of `int`.